tests/Db/Command/CommandTest.php

```diff
@@ -459,22 +460,38 @@ public function testDropView(): void
         );
     }
 
-    public function testDropTable(): void
+    public static function dataDropTable(): iterable
```

Comment:
Data providers in `db` packages are in separate classes like `\Yiisoft\Db\Tests\Provider\CommandProvider`
To reduce code in the test classes.

Reply:
done